curriculum/challenges/english/01-responsive-web-design/learn-intermediate-css-by-building-a-picasso-painting/step-089.md

```diff
@@ -381,3 +382,13 @@ body {
 --fcc-editable-region--
 
 ```
+
+# --solutions--
+
+```html
+// TODO: Add solution once steps 029 and 070 is fixed!
```

Comment:
Now that this is fixed, are you going to add the solutions? Or, shall I?

Reply:
I can push them up